communities/asu-phoenix/community-impact-proposal.md

```diff
@@ -0,0 +1,123 @@
+## Summary of the Community's Needs
+
+
+The community assessment helped assist with the fact that there are resources at ASU, but it seems to be disconnected. The community, especially freshmen who come to the campus are overwhelmed with the size and what the school has to offer that they never quite find what they were looking for. We spent the last semester organizing a hackathon to welcome beginners into the tech field but our organization was not the only one to do this, 2 other hackathons arised just 2 weeks apart from each other. After addressing the concern of the organizational disconnect with Joe, a more solidified idea started forming. There is a way to fortify the community and it is to connect beginners/freshmen alike to join and take part in the community to have them one day pass on the baton to the next batch. If we can bring the community together, organizations may soon follow and support each other for a thriving community. 
```

Comment:
Wow that density of events 😓 
Another thing here that I hadn't realised in our chat was the size of the school and amount of non-hackathon clubs. I think you're right that being overwhelmed would be a huge factor; providing resources to help them navigate all the options and groups they could join could be really valuable